articles/getting_to_know/whatis/graphics/WhatIs_Camera.md

```diff
@@ -0,0 +1,311 @@
+                    Vector3.Zero,
+                    // The rotation of the view, the direction of the upper edge of the camera.
+                    Vector3.Up);
+
+// The frustrum of the camera
+Matrix projectionMatrix = Matrix.CreatePerspectiveFieldOfView(
+    // The width of the camera view.
+    MathHelper.PiOver4,
+    // The height of the camera view.
+    GraphicsDevice.Viewport.AspectRatio,
+    // The near plane, when to start rendering objects from.
+    1.0f,
+    // The far plane, when to stop rendering objects from.
+    300.0f);
+```
+
+## [2D drawing](../../howto/graphics/HowTo_Draw_A_Sprite.md)
+
+Now to break one illusion, 2D drawing (without all the camera setup) is still 3D drawing but from a fixed point of view with all content drawn at the same physical depth.
+
+The [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch) class does a very clever job of setting up this fixed view behind the scenes and also provides some basic ordering/laying of content as it is drawn into view.  Each batch is sent to the graphics card collectively and then processed in order from first to last, each subsequent draw being on top of the last.
+
+> [!NOTE]
+> Texture / Font drawing is effectively like placing pieces of paper on a table, one on top of another until you are done.  What can be seen once all the paper has been placed is the final view.
+
+An example 2D drawing method:
+
+```csharp
+void DrawTexture(SpriteBatch spriteBatch, Texture2D spriteTexture, Vector2 spritePosition, Color spriteColor)
+{
+    spriteBatch.Begin();
+    spriteBatch.Draw(spriteTexture, spritePosition, spriteColor);
+    spriteBatch.End();
+}
+```
+
+The functionality does provide some techniques to sort what is drawn, provide some additional depth and layers, as well as blending options to effectively merge what is drawn.
+
+## [3D Drawing](../../howto/graphics/HowTo_RenderModel.md)
+
+When drawing 3D content in a scene, the Camera's current `View` and `Projection` are what are fed in to an [Effect](xref:Microsoft.Xna.Framework.Graphics.Effect) that draws the 3D content, together with a [Transformational Matrix](../../howto/graphics/HowTo_TransformPoint.md) (`World` position and rotation of the model) multiplied by the same `World` matrix used by the camera (as they are both part of the same world).
+
+An Example model drawing method:
+
+```csharp
+/// <summary>
+/// Draw the model, a model can have multiple meshes, so loop.
+/// </summary>
+/// <param name="aModel">The Model to draw.</param>
+/// <param name="aWorld">The World matrix from the camera or shared world matrix.</param>
+/// <param name="aView">The camera view in the world.</param>
+/// <param name="aProjection">The camera projection in the world.</param>
+void DrawModel(Model aModel, Matrix aWorld, Matrix aView, Matrix aProjection)
+{
+    //Copy any parent transforms
+    Matrix[] transforms = new Matrix[ aModel.Bones.Count ];
```

Comment:
Do we need this code here ? This is realted to bones and animations, not sure if its needed

Can we not use the `model.Draw` method to make things simpler?

Reply:
I suppose for this article that would make sense, I will update

Reply:
Sample uploaded to the Samples repo, once merged, I'll update the links in this doc and remove the zip @dellis1972 
https://github.com/MonoGame/MonoGame.Samples/pull/80

Reply:
Darn, need to still update this in the published code

Reply:
Updated sample with simplier call, but leaving in the original code for reference.